packages/replace/README.md

```diff
@@ -179,6 +179,19 @@ replace({
 });
 ```
 
+### `sourceMap` or `sourcemap`
```

Comment:
please place this in alphabetical order (before `values`)

Reply:
@shellscape Thanks for the feedback, order fixed